### PR TITLE
Fix for issue #1313

### PR DESCRIPTION
**Pull request by leon575777642**
#### Related Issue

#1313 

#### Types of changes

- [x] Bug fix (change which fixes an issue)
- [ ] New feature (change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

#### Checklist:

- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [x] I have added tests to cover my changes
- [x] All new and existing tests passed

### CODE REVIEW

File: utils/fasm/src/fasm.cpp [hunk 1]
```diff
@@ -679,7 +679,13 @@ void FasmWriterVisitor::output_fasm_mux(std::string fasm_mux_str,
 
     bool have_prefix = false;
     std::string clb_prefix;
-    find_clb_prefix(mux_input_pin->parent_node, &have_prefix, &clb_prefix);
+
+    if (mux_input_pin->port->type == IN_PORT) {
+        find_clb_prefix(mux_input_pin->parent_node, &have_prefix, &clb_prefix);
+    } else {
+        VTR_ASSERT(mux_input_pin->port->type == OUT_PORT);
+        find_clb_prefix(mux_input_pin->parent_node->parent_pb_graph_node, &have_prefix, &clb_prefix);
```
Comment (litghost) [inline utils/fasm/src/fasm.cpp:687]: This needs a comment

Comment (leon575777642) [reply utils/fasm/src/fasm.cpp:694]: Added.

**Comment by kmurray** [comment]
@leon575777642 / @litghost Is this fix now ready to merge?

**Comment by leon575777642** [comment]
> @leon575777642 / @litghost Is this fix now ready to merge?

Yes. Please review the changes.

**Comment by kmurray** [comment]
Thanks!

**Pull request merged**